articles/install-guide/vs-code.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Installing the Q# development environment for VS Code
+author: chgranad
+uid: microsoft.quantum.install.vs-code
+ms.author: christopher.granade@microsoft.com
+ms.date: 7/31/2018
+ms.topic: article
+---
+
+# Installing the Quantum Development Kit for Visual Studio Code #
+
+The Quantum Development Kit uses the .NET Core SDK (2.0 or later) to make it easy to create, build, and run Q# projects from the command line.
+We recommend using the Quantum Development Kit together with Visual Studio Code.
+
+1. Please see [Getting started with .NET](https://www.microsoft.com/net/learn/get-started) for how to download and install the .NET Core SDK.
+
+2. Once the .NET Core SDK is installed, run the following command in your favorite command line (e.g.: PowerShell or Bash) to download the latest templates for creating new Q# applications and libraries:
+   ```Bash
+   $ dotnet new -i "Microsoft.Quantum.ProjectTemplates::0.2.1806.3001-preview"
+   ```
+
+3. Go to the [Visual Studio Code website](https://code.visualstudio.com/).
+
+4. Pick the download for your platform and follow the installation prompts.
+
+5. Once Visual Studio Code is installed, go to the [Microsoft Quantum Development Kit for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=quantum.quantum-devkit-vscode) extension on the Visual Studio Marketplace and press Install.
+
+> [!NOTE]
+> When you run Visual Studio Code for the first time, you may be prompted to install Git if you have not already done so.
+> Git is not required to use the Quantum Development Kit, but works great for managing and sharing Q# projects, and for contributing back to the Q# libraries and samples.
+
+> [!TIP]
+> You can also install the Quantum Development Kit extension for Visual Studio Code from the *Go to file...* palette.
+> Press **Ctrl+P** or **⌘+P** from within Visual Studio Code, type or paste `ext install quantum.quantum-devkit-vscode`, and press **Enter**.
+
+You should now have the Quantum Development Kit installed and ready to use in your own applications and libraries.
+
+## Building the Quantum Samples and Libraries
```

Comment:
As with VS installation. Move this to its own "Validating" section, instead of a sub-section of "installing"